docs/personalization/cross_content_type_reco.md

```diff
@@ -0,0 +1,45 @@
+# Cross content type recommendations
+
+Cross content type option is used to combine best recommendation items from different content types.
+It applies to scenarios which have more than one output type configured.
+
+## Configure scenario
+
+To get multiple output types in the recommendation request, perform the following:
+
+1. Go to **Scenarios**.
+1. Click the **Edit** icon next to the scenario you want to set cross content type recommendations.
+1. From the **Output type** multiselect field, select the types you want to get recommendations for in the request.
+    It contains a list of all content types exported for the specified customer ID.
+1. Next, go to Preview Scenario.
+1. From the **Output type** drop-down, select **All**. 
+
+The **All** option automatically appears after content types are selected in the scenario settings.
```

Comment:
It seems that if one has selected several output types in the edit phase, the "All" option is selected by default when you go to. preview, so there is no need to select it. You could say something like...

``` suggestion
1. From the **Output type** drop-down, select the outputs for which you want to generate the preview. 
```

And then, the sentence about the "All" option makes more sense:

"After you select content types in the scenario settings, the **All** option automatically appears here and, by default, it is selected."

However, on a second thought, it is describing the inner workings of the system, so why not simply:

"By default, the **All** option is selected."